test/crawler/keyword/keywords_test.exs

```diff
@@ -0,0 +1,52 @@
+defmodule Crawler.Keyword.KeywordsTest do
+  use Crawler.DataCase, async: true
+
+  alias Crawler.Keyword.Keywords
+  alias Crawler.Keyword.Schemas.Keyword
+
+  describe("list_keywords/1") do
```

Comment:
For some files, you are adding (), for some you don't. I think it is better to be consistent. In client projects, we usually use it without ().

```suggestion
  describe "list_keywords/1" do
```

Reply:
Fixed in [71537f0](https://github.com/vnntsu/elixir-ic/pull/36/commits/71537f0e5eb384be7d8ce4bd12a48e3b9856bc83)